notation/lib/variables.ts

```diff
@@ -0,0 +1,39 @@
+import * as taskLib from 'azure-pipelines-task-lib/task';
+
+// get artifact references from input or from previous docker push task
+// through "RESOURCE_URIS" variable.
+export function getArtifactReferences(): string[] {
+    let artifactRefs = []
+    const artifactRefsInput = taskLib.getInput('artifactRefs', false);
+    if (!artifactRefsInput) {
+        artifactRefs = getArtifactReferencesFromDockerTask();
+        if (artifactRefs.length === 0) {
+            throw new Error('Artifact references are not specified');
+        }
```

Comment:
How about moving this check after this `if-else` statement and before line 17?

Reply:
Updated.